backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/Schema.kt

```diff
@@ -1,12 +1,15 @@
 package app.ehrenamtskarte.backend.regions.database
 
+import app.ehrenamtskarte.backend.projects.database.Projects
 import org.jetbrains.exposed.dao.IntEntity
 import org.jetbrains.exposed.dao.IntEntityClass
 import org.jetbrains.exposed.dao.id.EntityID
 import org.jetbrains.exposed.dao.id.IntIdTable
+import org.jetbrains.exposed.sql.ReferenceOption
 
 object Regions : IntIdTable() {
-    val regionIdentifier = varchar("regionIdentifier", 5).uniqueIndex() // 5-stelliger Kreisschlüssel
+    val projectId = reference("projectId", Projects, onDelete = ReferenceOption.CASCADE)
```

Comment:
Should I set `onDelete = ReferenceOption.CASCADE` for cards as well?

Reply:
Since we decided not to delete projects and regions anymore this can be removed again.